_docs/setup/kubernetes/mesh-expansion.md

```diff
@@ -201,6 +204,41 @@ or other mesh expansion machines.
    ... html content ...
 ```
 
+Check that the processes are running:
+```bash
+ps aux |grep istio
+```
+```
+root      6941  0.0  0.2  75392 16820 ?        Ssl  21:32   0:00 /usr/local/istio/bin/node_agent --logtostderr
+root      6955  0.0  0.0  49344  3048 ?        Ss   21:32   0:00 su -s /bin/bash -c INSTANCE_IP=10.150.0.5 POD_NAME=demo-vm-1 POD_NAMESPACE=default exec /usr/local/bin/pilot-agent proxy > /var/log/istio/istio.log istio-proxy
+istio-p+  6999  0.0  0.0  45236  4632 ?        Ss   21:32   0:00 /lib/systemd/systemd --user
+istio-p+  7000  0.0  0.0  61192  1924 ?        S    21:32   0:00 (sd-pam)
+istio-p+  7016  0.0  0.1 215172 12096 ?        Ssl  21:32   0:00 /usr/local/bin/pilot-agent proxy
+istio-p+  7094  4.0  0.3  69540 24800 ?        Sl   21:32   0:37 /usr/local/bin/envoy -c /etc/istio/proxy/envoy-rev1.json --restart-epoch 1 --drain-time-s 2 --parent-shutdown-time-s 3 --service-cluster istio-proxy --service-node sidecar~10.150.0.5~demo-vm-1.default~default.svc.cluster.local
```

Comment:
Can we remove some of irrelevant info here?

Reply:
was curious was (sd-pam) was but the rest is relevant no ?

Reply:
done